pkg/preflight/utils.go

```diff
@@ -24,6 +24,11 @@ func IsAzure() bool {
 	return Instance().ProviderName() == string(cloudops.Azure)
 }
 
+// IsAWS() returns whether the cloud environment is running on AWS
+func IsAWS() bool {
```

Comment:
Why not put this in `drivers/storage/portworx/util/util.go` where the rest of the platform checks are?

Reply:
This is where all the Instance() checks are.  The AWS provider is determined using preflight and k8s properties.   Its not determined by a spec like how we determine Pure and Vsphere.    So I have it here.